refinery/config/settings/base.py

```diff
@@ -600,3 +620,5 @@ def get_setting(name, settings=local_settings):
 
 # To avoid Port conflicts between LiveServerTestCases http://bit.ly/2pb64KN
 os.environ["DJANGO_LIVE_TEST_SERVER_ADDRESS"] = "localhost:10000-12000"
+
+DJANGO_DOCKER_ENGINE_URL_ROOT = "visualizations"
```

Comment:
May be `DJANGO_DOCKER_ENGINE_BASE_URL` to keep the nomenclature consistent with other similar uses? Also, why not `/visualizations` since it is a base URL?